recipe/meta.yaml

```diff
@@ -10,25 +10,28 @@ source:
   md5: fff8a7d7f580b04bacc2ffe7ddf41133
 
 build:
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - ssl_match_hostname  # [py<35]
+    - ssl_match_hostname   # [py<35]
```

Comment:
Reading the `setup.py`:
```python
if sys.version_info < (2, 7, 9):
  install_requires.append('backports.ssl_match_hostname')
```
Welp, I don't think we've ever built an older python than that, but `[py2k]` would not be unreasonable, I imagine. Waiting to see if the tests run, as is.